packages/opentelemetry-core/src/context/propagation/B3MultiPropagator.ts

```diff
@@ -138,4 +138,14 @@ export class B3MultiPropagator implements TextMapPropagator {
     }
     return context;
   }
+
+  fields() {
```

Comment:
Would be nice if you can add test for the same.

Reply:
Done